doc/howtos/building-images.rst

```diff
@@ -256,6 +284,52 @@ reconfigure it so that it matches ``ostro-image-swupd-dev``::
     OSTRO_IMAGE_NOSWUPD_EXTRA_FEATURES_append = "${OSTRO_IMAGE_FEATURES_DEV}"
     OSTRO_IMAGE_NOSWUPD_EXTRA_INSTALL_append = "${OSTRO_IMAGE_INSTALL_DEV}"
 
+Adding a Custom Layer in Ostro OS
+---------------------------------
+
+There are two practical options for adding a custom layer into your Ostro OS build.
+The first keeps your custom layer independent from ostro-os:::
+
+   $ git clone <your-custom-layer>
+   $ source oe-init-build-env
+   $ bitbake-layers add-layer $LAYERDIR     # add the custom layer path in OSTRO_LAYERS
+   $ bitbake-layers show-layers             # verify bitbake sees the layer
+
+A second approach is to add your custom layer by pulling it into the Ostro build with the ``combo-layer`` script::
+
+   $ git clone https://github.com/ostroproject/ostro-os.git     # or clone using git: if you've got your ssh keys setup
+   $ cd ostro-os
+   $ $EDITOR conf/combo-layer.conf        # add the layer repo details in the combo-layer.conf file
+   $ $EDITOR conf/combo-layer-local.conf  # and match those changes and repo local path
+   $ scripts/combo-layer update <your-custom-layer>
+   $ source oe-init-build-env
+   $ bitbake-layers add-layer $LAYERDIR   # add the custom layer path in OSTRO_LAYERS
+   $ bitbake-layers show-layers           # verify bitbake sees the layer
+
+Here's a practical example of using this second approach to add Intel's librealsense library
+to the Ostro OS image.  We'll start by adding the librealsense repo details to the ``combo-layer.conf``
+and ``combo-layer-local.conf`` files:::
+
+   [meta-intel-librealsense]
+   src_uri = git@github.com:IntelRealSense/meta-intel-librealsense.git
+   dest_dir = meta-intel-librealsense
+   hook = conf/combo-layerhook-generic.sh
+   branch = master
+   last_revision =
+
+Then continue with the second approach details above:::
+
+   $ cd ..
+   $ git clone git@github.com:IntelRealSense/meta-intel-librealsense.git
+   $ cd ostro-os
+   $ scripts/combo-layer update -n meta-intel-librealsense
+   $ bitbake-layers add-layer $REALSENSE_LAYERDIR
+   $ bitbake-layers show-layers
+
+And with that, we're ready to do a build:::
+
+   $ bitbake -k ostro-image-noswupd       # for example
```

Comment:
@gvancuts @mythi @uhofemei Can one of you give me a paragraph to work with here.